pyomo/core/tests/examples/test_kernel_examples.py

```diff
@@ -64,8 +64,9 @@ def testmethod(self):
             if (not testing_solvers['ipopt','nl']) or \
                (not testing_solvers['mosek_direct','python']):
                 self.skipTest("Ipopt or Mosek is not available")
-        rc, log = pyutilib.subprocess.run([sys.executable,example])
-        self.assertEqual(rc, 0, msg=log)
+        result = subprocess.run([sys.executable,example],
+                                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        self.assertEqual(result.returncode, 0, msg=result.stdout.decode("utf-8"))
```

Comment:
Is it sufficient to only output the stdout is the test fails?  should we consider `stderr=subprocess.STDOUT`?

Reply:
Good question. I was recreating the capability of the call previously - which I assume was previously considered to be sufficient. I'm happy to change it if you disagree with previous design choices.